src/documentation/pipelines/test-your-pipeline.md

```diff
@@ -5,4 +5,263 @@ permalink: /documentation/pipelines/test-your-pipeline/
 ---
 # Test Your Pipeline
 
-> **Note:** There is an open JIRA issue to create this guide ([BEAM-901](https://issues.apache.org/jira/browse/BEAM-901)).
+* TOC
+{:toc}
+
+Testing your pipeline is a particularly important step in developing an effective data processing solution. The indirect nature of the Beam model, in which your user code constructs a pipeline graph to be executed remotely, can make debugging-failed runs a non-trivial task. Often it is faster and simpler to perform local unit testing on your pipeline code than to debug a pipeline's remote execution.
+
```

Comment:
Introduce the DirectRunner (with link to the page Melissa is adding) by name as something great for testing and local development?